linux-tkg/linux-tkg-config/generic-desktop-profile.cfg

```diff
@@ -0,0 +1,55 @@
+# linux59-TkG config file
```

Comment:
This could be changed to "# linux-tkg config file" I guess

Reply:
fixed